includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -318,4 +301,54 @@ private function getLangWithFallback( $lang ) {
 			'help' => 'http://ko.community.wikia.com/wiki/%ED%8A%B9%EC%88%98%EA%B8%B0%EB%8A%A5:%EB%AC%B8%EC%9D%98',
 		],
 	];
+
+	private $socialHrefs = [
+		'en' => [
+			'facebook' => 'https://www.facebook.com/getfandom',
+			'twitter' => 'https://twitter.com/getfandom',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "twitter" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)